test/suite/helper_tiered.py

```diff
@@ -85,4 +85,5 @@ def get_fs_config(storage_source, additional_conf = '', test_name = ''):
         fs_conf += additional_conf
         return fs_conf
     
-    return None
\ No newline at end of file
+    return None
+    
```

Comment:
Not sure why GitHub flags here as if there was a missing extra line, do you see it too?

Reply:
Yes, it was a space - fixed it. thanks for catching this one.